frontend/src/components/RegistrationComplete/RegistrationComplete.jsx

```diff
@@ -31,13 +33,15 @@ const Button = styled.button`
   font: Urbanist;
 `;
 
-export default function RegistrationComplete() {
+export default function RegistrationComplete({ selectedEvent }) {
   return (
     <div>
       <Container>
         <h1>Registered For</h1>
       </Container>
-      <div>{/* EventCard placeholder here */}</div>
+      <div>
+        <EventCard event={selectedEvent} />
+      </div>
```

Comment:
Looks like your changes from the last PR are in this one too. These changes should go away once we merge in your old PRs, but in the future make sure you are always branching off the most recent version of main